examples/kmm/shared/src/commonMain/kotlin/com/mongodb/realm/realmkmm/RealmModels.kt

```diff
@@ -0,0 +1,29 @@
+package com.mongodb.realm.realmkmm
+
+import io.realm.RealmObject
+
+class Person : RealmObject {
+    var name: String = "Foo"
+    var dog: Dog? = null
+}
+
+class Dog : RealmObject {
+    var name: String = ""
+    var age: Int = 0
+}
+
+// :code-block-start: landing-page-model
+class Frog : RealmObject {
+    var name: String = ""
+    var age: Int = 0
+    var species: String? = null
+    var owner: String? = null
+}
+// :code-block-end:
+
+// :code-block-start: quick-start-model
+open class Task : RealmObject {
```

Comment:
```suggestion
class Task : RealmObject {
```

`open` is no longer required.